### DISCUSSION TITLE
Workaround field for issue tracker?

### DISCUSSION THREAD

**Discussion by typesanitizer** [arxanas/git-branchless#227] [General]
The [GitHub Issues features page](https://github.com/features/issues) mentions:

> Extend issues with custom fields [Beta]
>
> Track metadata like iterations, priority, story points, dates, notes, and links. Add custom fields to project tables and edit from the issue sidebar.

Thoughts on adding a Workaround field to the issue tracker? For example, https://github.com/arxanas/git-branchless/issues/219 has a workaround where you can tweak include paths in the configuration. It seems like this field would be generally useful elsewhere too.

**Comment by arxanas** [comment]
Generally, I like to keep the usage of project management features to a minimum in my projects, because I notice I end up spending a lot of time on bookkeeping for no reason. Actually, I was thinking to remove the existing Projects tab, since I don't think it's adding a lot of value 😂. (It seems like I can actually use a "private" project on Github to track my own planned work.) If we end up getting a lot of users who get confused and need a workaround, then we can add it.

**Reply by arxanas** [reply]
Looking at it more, I think the "custom fields" are only applied to the Project view, not the issue tracker? So you wouldn't see the workaround field unless you clicked through to the corresponding Project.

**Reply by typesanitizer** [reply]
My bad, I misunderstood the wording to mean you could literally add a new field to an issue. Reducing project management stuff if it isn't working for you makes sense. 👍🏽
